modules/@apostrophecms/modal/ui/apos/mixins/AposEditorMixin.js

```diff
@@ -93,7 +93,7 @@ export default {
             };
           } catch (error) {
             await apos.notify(this.$t('apostrophe:errorEvaluatingExternalCondition', { name: field.name }), {
-              type: 'error',
+              type: 'danger',
```

Comment:
we can now use "danger" in e2e tests, let's revert it back to it

cf https://github.com/apostrophecms/cypress-tools/pull/22

Reply:
Oh yes, 'danger' makes the icon red with an exclamation mark, whereas 'error' just shows a grey empty icon, as if this type does not exist...

![image](https://user-images.githubusercontent.com/8301962/224662857-e1efad60-bea9-4c08-9e7d-cb67b9345171.png)

![image](https://user-images.githubusercontent.com/8301962/224663019-e171cb4e-e044-4e89-a271-faf8b7aaf30c.png)
![image](https://user-images.githubusercontent.com/8301962/224663069-d65f8509-219c-4e45-92a6-ac759ab31cd9.png)

![image](https://user-images.githubusercontent.com/8301962/224663347-51d2fad4-09b8-43ef-8422-92835bfb01cc.png)